src/v2/view-builder/ViewFactory.js

```diff
@@ -45,11 +48,12 @@ const VIEWS_MAPPING = {
   },
   'enroll-factor': {
     email: RequiredFactorEmailView, // TODO EnrollFactorEmailView is unimplemented
-    password: EnrollFactorPasswordView
+    password: EnrollFactorPasswordView,
```

Comment:
I guess we're not going to support WebAuthn enrollment on IdX for now, since we can do it on v1?

Reply:
yeah  I dint add here anything for enroll.